### ISSUE TITLE
Should CommComm own the Node.js Foundation website?

### ISSUE THREAD

**Issue by obensource** [nodejs/community-committee#316]
## A new initiative?
The [Foundation website](https://foundation.nodejs.org/) is out of date and broken in places. I'm not sure if there is an active maintainer, but should CommComm formally request to own it and make it our initiative?

### Inaccurate count:
<img width="229" alt="screen shot 2018-05-15 at 9 47 26 pm" src="https://user-images.githubusercontent.com/1910457/40097302-3ff1d98c-588a-11e8-900f-4f4fb3827c2b.png">

### Broken images
<img width="1270" alt="screen shot 2018-05-15 at 9 48 09 pm" src="https://user-images.githubusercontent.com/1910457/40097320-57cdf07c-588a-11e8-8b65-618e7d40e955.png">

**Comment by chowdhurian** [comment]
+1.

Very much worth discussing. Thank you for raising the issue, Ben! I believe the topic came up during a website redesign meeting/discussion. Ping @nodejs/website-redesign 👆🏽

Some action does need to be taken, and the redesign initiative is a good excuse to sort this out. Ideally, people working with the Foundation have the access and control they need to manage the site. And some parts of it need to stay current without their having to keep track of and update on their end.

Whatever action is recommended, people in the Foundation will need to be involved in the running of it in some capacity. It is their site after all! 🙂

**Comment by ladyleet** [comment]
@Tiriel Yes, agreed that y'all should focus on the site redesign at first and I'll work within the Node Foundation to find a solution to address these problems. Just like any website - we just need to be informed so we can fix immediately. 

The issues that @obensource brought up I just raised to get them fixed... 

I do believe there may be a few current limitations for the foundation website itself and its maintained by the foundation. Still working internally to find out more info for y'all! Thanks again for raising the issue. 

@hackygolucky love any "prior experience" thoughts on this. ;)

**Comment by Tiriel** [comment]
@ladyleet Don't get me wrong, I don't think we should prioritize one above the other, although that might become an issue too.

I was in fact mostly wondering about the "legal" aspects of the question, which it seems you are already enquiring about. Thanks for your quick answer anyway!

**Comment by WaleedAshraf** [comment]
It's not in Node.js Foundation on GitHub. I have seen multiple issues, where people create the ticket in [nodejs.org](https://github.com/nodejs/nodejs.org) site and confuse it with Foundation Website. And then someone tags a person from Foundation to fix the issue.

I'm +1 on moving it to Node.js Foundation on GitHub.

**Comment by mrhinkle** [comment]
We'll add a clearer way to add the reporting for Node.js Foundation website and the delineation between nodejs.org and foundation.nodejs.org. Its hosted on the on Wordpress for ease of administration and won't be moving to Github. Greg Wallace and Shawn Preissner will be glad to be responsive if there are issues that need to be addressed.  We'd be glad to work with anyone who wants to provide feedback and we'll drive to get necessary changes done more quickly if they aren't being addressed. Well talk it over and get back to you to see if our suggestions make sense for everyone.

**Comment by obensource** [comment]
Thank you @mrhinkle! 🙌

**Comment by chowdhurian** [comment]
To remedy future inaccuracies, it may be helpful to project if we were to create and maintain an up-to-date and limited reference of information. This information should be specific to the numbers and information regularly requested for resources created by the Foundation. It should be limited; to minimize the burden of keeping it updated. 💚

I suggest this as a partial solution to any difficulties sourcing accurate and reliable high-level information about the Node.js project. ✅

**Comment by fhemberger** [comment]
Slightly off-topic: There's also a plug-in to sync Wordpress themes with GitHub, so at least we could sync the Foundation's website design with the upcoming nodejs.org relaunch: https://github.com/afragen/github-updater

**Comment by chowdhurian** [comment]
@fhemberger so using this WordPress plugin:
* the Foundation's theme could be built to match the website redesign and hosted on Github
* the Foundation would still have full control of content while the design of the website would match the redesign of nodejs.org.

Is my interpretation of this tool correct? 😅

**Comment by fhemberger** [comment]
@chowdhurian Exactly.

**Comment by WaleedAshraf** [comment]
Any update on this issue?

**Comment by WaleedAshraf** [comment]
+1 for discussing in the next meeting. Can we add someone from LF side?

**Comment by SarahKConway** [comment]
Happy to discuss this with CommComm at the next meeting. Looking forward to seeing the new website work from @amiller-gh.  Adding @kbarnard10 - our main marketing contact for the NF & JSF websites.

**Comment by amiller-gh** [comment]
We discussed this at the tail end of Website Redesign last week. After going through the site IA with @RRomoff we're going to try and find homes for all the content on the Foundation site we'd like to see carry over post-foundation-merge in the new site IA. We'll continue working with @SarahKConway and @kbarnard after drafting a plan for the content we're planning to port over. I'll open an issue with the site IA notes I took during the walkthrough in @nodejs/website-redesign and we'll continue work over there 🙂 I'll close this ticket once I open the other one with a cross-ref.
